pkg/selinux/selinux-client.js

```diff
@@ -101,8 +103,24 @@ client.init = function(statusChangedCallback) {
         );
     };
 
-    refreshInfo();
-    window.setInterval(refreshInfo, pollingInterval);
+    var polling = null;
+
+    function setupPolling() {
+        if (cockpit.hidden) {
+            window.clearInterval(polling);
+            polling = null;
+        } else if (polling === null) {
+            polling = window.setInterval(refreshInfo, pollingInterval);
+            refreshInfo();
+        }
+    }
+
+    cockpit.addEventListener("visibilitychange", setupPolling);
+    setupPolling();
+
+    /* The first time */
+    if (polling === null)
```

Comment:
I would want at least some information to be present when it jumps into view the first time. Even if it then changes once we get an update.